source/nodejs/adaptivecards-designer/src/card-designer.ts

```diff
@@ -1473,6 +1473,15 @@ export class CardDesigner extends Designer.DesignContext {
 
             this.setCardPayload(card, false);
             this.updateToolbar();
+
+            const undoSuccessPopup = document.createElement("p");
+            undoSuccessPopup.textContent = "Successfully undid your last change";
```

Comment:
what's message used in other product or other webside? will "Undo Complete" and "Redo Complete" be better?

Reply:
I couldn't find an example online, but I do like the shorter text